changelog/updates/2022-08-29-intel-microcode-20220809_p20220809.md

```diff
@@ -0,0 +1 @@
+- intel-microcode ([20220809](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/releases/tag/microcode-20220809))
```

Comment:
You would then probably want to delete the added files from `changelog/updates` and `changelog/security`.